manifest.json

```diff
@@ -20,7 +20,7 @@
 
   "content_scripts": [
     {
-      "matches": ["*://bitbucket.org/*/pull-requests/*"],
+      "matches": ["<all_urls>"],
```

Comment:
That won't work in current incarnation. The script will load for every page, which means it will measure time for every page and we don't want that.

Reply:
So I guess this would have to stay more or less open like this and we have to guard it in the script itself